packages/authentication/addon/templates/components/cardstack-authentication-launcher.hbs

```diff
@@ -1,8 +1,13 @@
 {{#if session.isAuthenticated}}
   {{#if network.iconUrl}}
     <div class="cardstack-rail-item cardstack-non-interactive">
-      <div class="cardstack-network-icon"
-           style={{css-url "background-image" network.iconUrl}} >{{network.name}}</div>
+      {{! template-lint-disable no-inline-styles }}
+      <div
+        class="cardstack-network-icon"
+        style={{css-url "background-image" network.iconUrl}}
+      >
```

Comment:
Maybe we should just disable `no-inline-styles` globally? Doesn't seem like a particularly useful rule to me 🤔

Reply:
Good idea.